diff -up aspell-0.60.6/prog/checker_string.cpp.gcc33 aspell-0.60.6/prog/checker_string.cpp
--- aspell-0.60.6/prog/checker_string.cpp.gcc33	2006-11-25 16:07:15.000000000 +0100
+++ aspell-0.60.6/prog/checker_string.cpp	2008-09-01 11:02:59.000000000 +0200
@@ -6,6 +6,7 @@
 
 #include <assert.h>
 
+#include <assert.h>
 #include "checker_string.hpp"
 #include "speller.hpp"
 #include "document_checker.hpp"
